src/auth_code.rs

```diff
@@ -13,7 +13,7 @@ use std::sync::Arc;
 use maybe_async::maybe_async;
 use url::Url;
 
-/// The [Authorization Code Flow](reference) client for the Spotify API.
+/// The [Authorization Code Flow][reference] client for the Spotify API.
```

Comment:
Is it the correct link?

Reply:
> Is it the correct link?

Hi, you can check out here https://docs.rs/rspotify/latest/rspotify/struct.AuthCodeSpotify.html, it points to a wrong url.

Reply:
Yes, it points to a wrong URL. But I think your updated one `[Authorization Code Flow][reference]` is also incorrect. The correct URL should be: https://developer.spotify.com/documentation/web-api/tutorials/code-flow

Reply:
> Yes, it points to a wrong URL. But I think your updated one `[Authorization Code Flow][reference]` is also incorrect. The correct URL should be: [developer.spotify.com/documentation/web-api/tutorials/code-flow](https://developer.spotify.com/documentation/web-api/tutorials/code-flow)

Yes, you're right, the old link would be redirected to the link you provide though. Shall I make an update?

Reply:
Yes, feel free to do that :)